docs.md

```diff
@@ -0,0 +1,40 @@
+## General Moments
+
+### Everything is a Knowledge Item. 
+#### KnowledgeItem 
+KnowledgeItem - abstract class, that is parental for all other items
+**Unique fields: _title, creation_time,  id: **
+
+
+### Information Classes 
+
+#### TextNote: Inherited from KnowledgeItem
+Storage class for text information. TextNote inherited from KnowledgeItem. 
+**Additional fields: string_text
+
+#### Comment: Inherited from KnowledgeItem
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam ultricies sit amet sapien in posuere. Maecenas accumsan dui ut diam molestie, sed consectetur libero eleifend. Suspendisse potenti
```

Comment:
Let's not use lorem ipsum in docs :)
Maybe replace it with something like "Description TBD" (TBD is a standard abbreviation for "to be defined/decided/determined")